getting-started.md

```diff
@@ -1,6 +1,6 @@
 # Getting started with Prodfiler
 
-Getting started with Prodfiler is easy: You create an account and a project, receive instructions
+Getting started with Prodfiler is easy: you create an account and a project, receive instructions
```

Comment:
Suggestion:
To get started simply create an account and a project.
We then provide instructions on how to deploy the agent.
About 30 minutes later the profiling data shows up in the UI.

Reply:
I simply changed the capitalized `Y` here because of grammar.
I would not change this as it was part of the original doc already.

Reply:
🤷🏻‍♂️ OK